framework/src/interfaces/DataFileInterface.C

```diff
@@ -40,6 +38,17 @@ std::string
 DataFileInterface<T>::getDataFileNameByName(const std::string & relative_path,
                                             const std::string * param) const
 {
+  bool issue_info = true;
+
+  // we do not want to issue the information for the face and neighbor materials
+  // because the same information has been outputted by the element material
+  auto mat_ptr = dynamic_cast<const Material *>(&_parent);
+  if (mat_ptr)
+  {
+    if (mat_ptr->isBoundaryMaterial() && mat_ptr->blockRestricted())
+      issue_info = false;
+  }
```

Comment:
this should go in the interface constructor and set a member variable.

Reply:
Yes.